client/src/telethon/_impl/client/events/queries.py

```diff
@@ -69,6 +70,21 @@ async def answer(
             )
         )
 
+    async def get_message(self) -> Optional[Message]:
+        """
+        Get the :class:`~telethon.types.Message` containing the button that was clicked.
+
+        If the message is too old and is no longer accessible, :data:`None` is returned instead.
+        """
+        peer_id_ = peer_id(self._raw.peer)
```

Comment:
Would prefer to just call this `pid`. It's harder to typo it for `peer_id`.

Reply:
fixed